docker-compose.yml

```diff
@@ -3,7 +3,17 @@
 version: "3"
 services:
   pyrdp:
-    build: .
+    image: gosecure/pyrdp:latest
+    # Slim image (no GUI pyrdp-player, no pyrdp-convert)
+    #image: gosecure/pyrdp:latest-slim
+    # Uncomment a build section if you want to build your own image
+    # Full image
+    #build: .
+    # Slim image (no pyrdp-player, no pyrdp-convert)
```

Comment:
is this still true? I thought the slim image could run pyrdp-player in headless mode

Reply:
```suggestion
    # Slim image (no pyrdp-player w/o --headless and no pyrdp-convert)
```

better?

Reply:
yeah, though I might be nitpicking too much but I find w/o to be confusing and unintuitive, I think shortcuts like that should be avoided in documentations

Reply:
```suggestion
    # Slim image (no pyrdp-player without --headless and no pyrdp-convert)
```
now?

Reply:
yes